Api/src/controllers/officeController.js

```diff
@@ -0,0 +1,86 @@
+// import {
+//   v4 as uuidv4
+// } from 'uuid';
+import fs from 'fs';
+
+import data from "../data.json" assert { type: "json" };
+
+let offices = data.Offices;
+
+// console.log(offices);
+
+export const getOffices = async (req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'application/json');
+  res.send((data.Offices));
```

Comment:
Incorrect return again! 

Self-review should have caught this. This either means you didn't do self-review before reopening the PR or that you didn't do it "detail-y" enough.

Reply:
While you're at it, ensure your Entities meet the Entity Specification as well, because I have not checked those yet. The API Endpoint Specification alone has take more than a week of review cylces. which is really not good.

Reply:
You're right, I later changed it in the officeRoute file and didn't remove it here. But I've refactored it.

Reply:
To be honest, it has taken so long because I've been doing it wrongly, and it's all my fault as my NodeJS skills wasn't so good. However, I've been able to improve thus far, and I believe it'll only get better as the project progresses.